tools/internal_ci/linux/grpc_psm_performance_gke_experiment.sh

```diff
@@ -0,0 +1,144 @@
+TEST_INFRA_VERSION=$(git describe --tags "$(git rev-list --tags --max-count=1)")
+popd
+
+# PSM tests related ENV
+PSM_IMAGE_PREFIX=gcr.io/grpc-testing/e2etest/runtime
+PSM_IMAGE_TAG=${TEST_INFRA_VERSION}
+
+# Build test configurations.
+buildConfigs() {
+    local -r pool="$1"
+    local -r table="$2"
+    local -r type="$3"
+
+    shift 3
+    tools/run_tests/performance/loadtest_config.py "$@" \
+        -t ./tools/run_tests/performance/templates/loadtest_template_psm_"${type}"_prebuilt_all_languages.yaml \
+        -s driver_pool="${DRIVER_POOL}" -s driver_image= \
+        -s client_pool="${pool}" -s server_pool="${pool}" \
+        -s big_query_table="${table}" -s timeout_seconds=900 \
+        -s prebuilt_image_prefix="${PREBUILT_IMAGE_PREFIX}" \
+        -s prebuilt_image_tag="${UNIQUE_IDENTIFIER}" \
+        -s psm_image_prefix="${PSM_IMAGE_PREFIX}" \
+        -s psm_image_tag="${PSM_IMAGE_TAG}" \
+        -a ci_buildNumber="${KOKORO_BUILD_NUMBER}" \
+        -a ci_buildUrl="${CLOUD_LOGGING_URL}" \
+        -a ci_jobName="${KOKORO_JOB_NAME}" \
+        -a ci_gitCommit="${GRPC_GITREF}" \
+        -a ci_gitCommit_java="${GRPC_JAVA_GITREF}" \
+        -a ci_gitActualCommit="${KOKORO_GIT_COMMIT}" \
+        --prefix="${LOAD_TEST_PREFIX}" -u "${UNIQUE_IDENTIFIER}" -u "${pool}" -u "${type}"\
+        -a pool="${pool}" \
+        --category=psm \
+        --allow_client_language=c++ --allow_server_language=c++ \
+        -o "psm_${type}_loadtest_with_prebuilt_workers_${pool}.yaml"
+}
+
+buildConfigs "${WORKER_POOL_8CORE}" "${BIGQUERY_TABLE_8CORE}" "proxied" -a "queue"="${WORKER_POOL_8CORE}-proxied"  -l c++ -l java --client_channels=8 --server_threads=16 --offered_loads 5000 6000
+buildConfigs "${WORKER_POOL_8CORE}" "${BIGQUERY_TABLE_8CORE}" "proxyless" -a "queue"="${WORKER_POOL_8CORE}-proxyless" -l c++ -l java --client_channels=8 --server_threads=16 --offered_loads 5000 6000 
+
+# Delete prebuilt images on exit.
+deleteImages() {
+    echo "deleting images on exit"
+    ../test-infra/bin/delete_prebuilt_workers \
+    -p "${PREBUILT_IMAGE_PREFIX}" \
+    -t "${UNIQUE_IDENTIFIER}"
+}
+trap deleteImages EXIT
+
+# Build and push prebuilt images for running tests.
+time ../test-infra/bin/prepare_prebuilt_workers \
+    -l "cxx:${GRPC_CORE_GITREF}" \
+    -l "java:${GRPC_JAVA_GITREF}" \
+    -p "${PREBUILT_IMAGE_PREFIX}" \
+    -t "${UNIQUE_IDENTIFIER}" \
+    -r "${ROOT_DIRECTORY_OF_DOCKERFILES}"
+
+# Run tests.
+time ../test-infra/bin/runner \
+    -i "psm_${PSM_PROXIED_TEST}_loadtest_with_prebuilt_workers_${WORKER_POOL_8CORE}.yaml" \
+    -i "psm_${PSM_PROXYLESS_TEST}_loadtest_with_prebuilt_workers_${WORKER_POOL_8CORE}.yaml" \
+    -i "psm_${PSM_PROXIED_TEST}_loadtest_with_prebuilt_workers_${WORKER_POOL_32CORE}.yaml" \
+    -i "psm_${PSM_PROXYLESS_TEST}_loadtest_with_prebuilt_workers_${WORKER_POOL_32CORE}.yaml" \
+    -log-url-prefix "${LOG_URL_PREFIX}" \
+    -polling-interval 5s \
+    -delete-successful-tests \
+    -annotation-key "queue" \
```

Comment:
The runner prints the value of annotation key before polling interval, so I would add this line right before polling interval.